packages/common/crypto-names/package.json

```diff
@@ -6,11 +6,19 @@
   "bugs": "https://github.com/dxos/dxos/issues",
   "license": "MIT",
   "author": "DXOS.org",
-  "main": "dist/lib/node/index.cjs",
-  "browser": {
-    "./dist/lib/node/index.cjs": "./dist/lib/browser/index.mjs"
+  "exports": {
+    ".": {
+      "browser": "./dist/lib/browser/index.mjs",
+      "node": {
+        "default": "./dist/lib/node/index.cjs"
+      },
+      "types": "./dist/types/src/index.d.ts"
+    }
+  },
+  "types": "dist/types/src/index.d.ts",
+  "typesVersions": {
+    "*": {}
```

Comment:
seen this in a couple packages, is it necessary for something even though it's empty?

Reply:
Not necessary but not harmful, I just didn’t want to complicate the tool code to handle this case

Reply:
If it’s too annoying I can add a case to remove it